apple/testing/default_runner/simulator_creator.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
```

Comment:
This file is missing a license header.

Reply:
hrm do we really need that? some BUILD files don't have it for example

Reply:
I don't know but BUILD files don't seem to need license header, from what I've seen in most Bazel repos (maybe they're not consider source code?)

Reply:
added it